tools/labs/templates/kernel_profiling/4-bio/bio.c

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
Remove comment lines.

This gives warnings at build time:
```
/home/razvan/school/so2/linux.git/tools/labs/skels/./kernel_profiling/4-bio/bio.c:93:28: warning: passing argument 2 of ‘kthread_create_on_node’ makes pointer from integer without a cast [-Wint-conversion]
   kthread_run(my_thread_f, i, "%skwriterd%d", "my", i);
                            ^
./include/linux/kthread.h:28:35: note: in definition of macro ‘kthread_create’
  kthread_create_on_node(threadfn, data, NUMA_NO_NODE, namefmt, ##arg)
                                   ^~~~
/home/razvan/school/so2/linux.git/tools/labs/skels/./kernel_profiling/4-bio/bio.c:93:3: note: in expansion of macro ‘kthread_run’
   kthread_run(my_thread_f, i, "%skwriterd%d", "my", i);
   ^~~~~~~~~~~
./include/linux/kthread.h:11:21: note: expected ‘void *’ but argument is of type ‘int’
 struct task_struct *kthread_create_on_node(int (*threadfn)(void *data),
                     ^~~~~~~~~~~~~~~~~~~~~~
```

Reply:
It's expected. I didn't want to pass an address.

Reply:
Do a cast to prevent them from happening.